about.html

```diff
@@ -16,17 +16,21 @@
 <body>
     <header>
         <ul>
-            <li><img src="pics/icon/facebook.png" alt="Facebook"></li>
-            <li><img src="pics/icon/twitter.png" alt="Twitter"></li>
+            <li><img id="face" src="pics/icon/facebook.png" alt="Facebook"></li>
+            <li><img id="twi" src="pics/icon/twitter.png" alt="Twitter"></li>
             <li>English</li>
             <li>My page</li>
             <li>Logout</li>
         </ul>
     </header>
 
     <div class="menuNav">
-        <nav id="navBar" class="navBar">     
-            <a href="about.html" class="about">About</a>
+
+        <button class="campaign">CC Campaign</button>
+
+        <nav id="navBar" class="navBar">   
+            <a href="index.html" id="home">Home</a>  
+            <a href="about.html" id="about">About</a>
             <a href="" class="program">Program</a>
```

Comment:
- 🙏Please ensure that the correct background color is used in the 📱mobile size screen, as specified in the original design. This adjustment will enhance the appearance of your project.

Your design:
![image](https://github.com/Dachrono/capstone/assets/63445083/228c8387-7d02-42c7-92f1-fbe08eac905c)
Original design:
![image](https://github.com/Dachrono/capstone/assets/63445083/d13789a2-b409-40b8-9766-5a31214bb51c)

Reply:
Solved, tky for you advise